Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptDeleteController.java

```diff
@@ -110,5 +114,13 @@ public String confirmlDelete(@PathVariable("id") String id, ModelMap model) {
 		model.addAttribute("result", foundConcepts);
 		return "/auth/conceptlist/concepts";
 	}
-
+	
+    @RequestMapping(value = "auth/conceptlist/deleteconcepts/{id}", method = RequestMethod.GET)
+    public String deleteConcept(@PathVariable("id") String id, ModelMap model) {
+        ConceptEntry concept = conceptManager.getConceptEntry(id);
+        deleteConceptManager.setDelete(concept);
+        conceptManager.storeModifiedConcept(concept);

```

Comment:
you would simply call
{code}
conceptManager.deleteConcept(id);
{code}

the conceptManager would retrieve the concept, set deleted to true and then store the concept.